features/firecracker/info.yaml

```diff
@@ -3,3 +3,4 @@ type: platform
 features:
   include:
     - server
+  exclude: [ _selinux ]
```

Comment:
exclude does not properly work yet, just skip the test instead

Reply:
Thanks, jeah just noticed it doesn't work